components/onboarding_tasks/onboarding_tasks_manager.tsx

```diff
@@ -113,6 +116,11 @@ export const useTasksList = () => {
         delete list.START_TRIAL;
     }
 
+    if (!isUserFirstAdmin && !isUserAdmin) {
+        delete list.VISIT_SYSTEM_CONSOLE;
+        delete list.START_TRIAL;
+    }
```

Comment:
Can we add test cases (either unit or cypress tests) to cover when the task list should show/not show for admin and end users?